.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: android
+
+jdk:
+ - oraclejdk7
+ - oraclejdk8
+
+matrix:
+  allow_failures:
+    - jdk: oraclejdk7
```

Comment:
@zhangkun83 @ejona86 I honestly rather not do that since you have to now hard code jvm args for the build. The build has a target/source of Java 1.7(no oracle support anymore) and uses jdk8.

I think it would be best if I just removed the jdk7 part. JDK8 automatically handles memory usage during builds.